plugins/VisitsSummary/Controller.php

```diff
@@ -106,6 +107,17 @@ public function getEvolutionGraph(array $columns = array(), array $defaultColumn
             'avg_time_generation'
         );
 
+        $period = Common::getRequestVar('period', false);
+
+        if (SettingsPiwik::isUniqueVisitorsEnabled($period)) {
+            // add number of unique visitors if available
+            $selectableColumns = array_merge(
+                array($selectableColumns[0]),
+                array('nb_uniq_visitors'),
+                array_slice($selectableColumns, 1)
```

Comment:
This may be a bit fragile. Maybe we could instead just do like

```php
$selectableColumns = array('nb_visits');
if (SettingsPiwik::isUniqueVisitorsEnabled($period)) {
   $selectableColumns [] = 'nb_uniq_visitors';
}
$selectableColumns [] = 'nb_users';
$selectableColumns [] = '...';
```

might be a little better to read and understand